tests/test_shell_zprofile_synthetic_login.py

```
@@ -0,0 +1,115 @@
+
+    return (True, "")
+
+
+def main() -> int:
+    root = Path(__file__).resolve().parents[1]
+    wrapper_dir = root / "Resources" / "shell-integration"
+    if not (wrapper_dir / ".zshenv").exists():
+        print(f"SKIP: missing wrapper .zshenv at {wrapper_dir}")
+        return 0
+
+    base = Path("/tmp") / f"cmux_zprofile_synthetic_login_{os.getpid()}"
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)